scripts/extensions/sams/src/notifications/index.ts

```diff
@@ -0,0 +1,30 @@
+import {superdeskApi} from '../apis';
+import {onSetCreated, onSetUpdated, onSetDeleted} from './sets';
+
+const websocketNotificationMap: {[key: string]: (event: any) => void} = {
+    'sams:set:created': onSetCreated,
+    'sams:set:updated': onSetUpdated,
+    'sams:set:deleted': onSetDeleted,
+};
+
+let websocketDeregistrationArray: Array<() => void> = [];
+
+export function registerWebsocketNotifications() {
+    if (websocketDeregistrationArray.length > 0) {
+        return;
```

Comment:
Add a comment here as to why we're returning early without registering any listeners